CHANGELOG.md

```diff
@@ -6,9 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Update nginx to 1.17.10 #32
```

Comment:
Please add a short description of the new attributes location and how it affects the actual creation of services with the new ces-conf version.